<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopulationStructureSetupDialog</class>
 <widget class="QDialog" name="PopulationStructureSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSocStatusClass">
     <property name="text">
      <string>Класс соц.статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbSocStatusClass</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>&amp;Дата</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Во&amp;зраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblAddressOrgStructureType">
     <property name="text">
      <string>Адрес</string>
     </property>
     <property name="buddy">
      <cstring>cmbAddressOrgStructureType</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="cmbAddressOrgStructureType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проживание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Регистрация или проживание</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прикрепление</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Регистрация или прикрепление</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Проживание или прикрепление</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Регистрация, проживание или прикрепление</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="chkDetalizeByHouses">
     <property name="text">
      <string>Детализация по домам</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="4" column="1">
    <widget class="CSocStatusComboBox" name="cmbSocStatusClass"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="text">
      <string>Тип соц.статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbSocStatusType</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>107</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CSocStatusComboBox</class>
   <extends>QComboBox</extends>
   <header>Registry/SocStatusComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbAddressOrgStructureType</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PopulationStructureSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PopulationStructureSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
